css/style.css

```diff
@@ -236,98 +236,201 @@ body {
     margin-left: -50px;
 }
 
+.profile{
```

Comment:
the class name should have the name of the page first followed by a hyphen and then the class name element. for example:
profilePage_container profilePage_title etc. I think we talked about this in the beginning about making sure the class names reflect the page on which they belong to. I'm only going to comment here but please make the changes accoridngly to the other class names you created below.